File: algorithm/typescript/BOJ2558.ts
// https://www.acmicpc.net/problem/2558

import fs = require("fs");

const input: number[] = fs.readFileSync("/dev/stdin").toString().split("\n").map(Number);

const A: number = input[0];
const B: number = input[1];

console.log(A + B);

File: algorithm/java/src/BOJ2558.java
// problem : https://www.acmicpc.net/problem/2558

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class BOJ2558 {
    static int A, B;

    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        A = Integer.parseInt(br.readLine());
        B = Integer.parseInt(br.readLine());

        System.out.println(A + B);
    }
}
